requirements.txt

```diff
@@ -7,6 +7,6 @@ python-dateutil==2.6.0
 pytz==2016.7
 scipy==0.18.1
 six==1.10.0
-statsmodels==0.6.1
+statsmodels==0.8.0
```

Comment:
Can you make this change to `setup.py` as well? When rdtools is installed via pip, it uses `setup.py` to download all of the corresponding dependencies.